### PR TITLE
[DEV-2464] Feature: Add GIF support

### PR DESCRIPTION
**Pull request by riffbyte**
Added display for `gif` images through Uploadcare's `gif2video` delivery.

### CODE REVIEW

File: src/components/Uploadcare/Gif/lib/getGifPoster.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    const { imageDetails, filename, src, effects = [], width } = options;
+
+    const sizes = getSizes(options);
+
+    return prepareUploadcareUrl({
+        width: width || sizes.default,
+        imageDetails,
+        src,
+        filename,
+        effects: [effect.format('png'), effect.format('auto'), ...effects],
```
Comment (yuriyyakym) [inline src/components/Uploadcare/Gif/lib/getGifPoster.ts:16]: Does it make sense to apply both format effects?

File: src/components/Uploadcare/Gif/lib/getGifSources.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { Options, VideoExtension } from '../../../../types';
+import effect from '../../lib/effects';
+import getSizes from '../../lib/getSizes';
+import prepareUploadcareUrl from '../../lib/prepareUploadcareUrl';
+
+const VIDEO_FORMATS: VideoExtension[] = ['mp4', 'webm'];
```
Comment (yuriyyakym) [inline src/components/Uploadcare/Gif/lib/getGifSources.ts]: `webm` should probably go first as it has better compression

**Review by yuriyyakym — APPROVED** [review]
👍

File: src/components/Uploadcare/Gif/lib/getGifPoster.ts (hunk 1 above)
Comment (riffbyte) [reply src/components/Uploadcare/Gif/lib/getGifPoster.ts:16]: @yuriyyakym This is how it was done in the slate-renderer. Honestly, I'm not sure if it's totally correct 🤔

**Review by yuriyyakym — APPROVED** [review]

**Pull request merged**